sudoku/validate_sudoku.py

```diff
@@ -0,0 +1,82 @@
+""" validate sudoku board
+    input is a board
+    Return True if board is correct
+    Return False if board is incorrect
+"""
```

Comment:
This is documentation for the function and not the file. It should be inside the function. Please look up online how to do this properly so you follow the convention.

Reply:
comments fixed in code